core/components/molecules/emptyState/_tests_/EmptyState.test.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { EmptyState } from '@/index';
+import { EmptyStateProps as Props } from '@/index.type';
+import { testHelper, filterUndefined, valueHelper, testMessageHelper } from '@/utils/testHelper';
+
+const sizes: Props['size'][]  = ['large', 'small'];
+
+describe('EmptyState component', () => {
+  const mapper: Record<string, any> = {
+    size: valueHelper(sizes, { required: true, iterate: true }),
+  };
+
+  const testFunc = (props: Record<string, any>): void => {
+    const attr = filterUndefined(props) as Props;
+    it(testMessageHelper(attr), () => {
+      const { baseElement } = render(
+        <EmptyState
+          {...attr}
+        />
+      );
+      expect(baseElement).toMatchSnapshot();
+    });
+  };
+  testHelper(mapper, testFunc);
+});
+
+const exampleString = 'Design';
+
+describe('EmptyState component', () => {
+  it('Should have EmptyState class', () => {
+    const { getByTestId } = render(
+    <EmptyState title={exampleString} imageSrc={exampleString} description={exampleString} size="large"/>
```

Comment:
Title and description should be as per stories. Please refer the stories.
For eg. title: There's a problem loading this page.